frontend/src/views/Room.jsx

```diff
@@ -254,12 +253,15 @@ function Room() {
       });
 
       newRoom.on('ParticipantLeft', (p) => {
+        // Check if the participant who left the room was sharing screen
+        if (remoteStreamsRef.current.get(p.id)?.isSharingScreen) {
+          setIsSharingScreen(false);
+        }
```

Comment:
I wonder if avoiding the if statement would be a good idea.
Since it's just setting a state, to do it always, either way if it is sharing screen or not, basically saving a few operations every time someone leaves a room (avoiding the if, just setting a state).

But might not be as clear why it is being done. Either way, it's fine.